RELEASE_NOTES.md

```diff
@@ -1,5 +1,91 @@
+- `apt-cdrom` repositories are now skipped when parsing out the list of apt repositories
+- Faulty profiles are now reported instead of causing a crash
+- Errors are no longer logged to stdout with the `html2` reporter
+- macOS Big Sur is now correctly identified as macOS
+
+## New Resources
+
+### windows_firewall_profile
+
+The `windows_firewall_profile` allows you to `enable`, `disable`, or `configure` Windows Firewall profiles. For example you can now setup default actions and configure rules for the `Public` profile using this single resource instead of managing your own PowerShell code in a `powershell_script` resource:
+
+```ruby
+windows_firewall_profile 'Public' do
+  default_inbound_action 'Block'
+  default_outbound_action 'Allow'
+  allow_inbound_rules false
+  display_notification false
+  action :enable
+end
+```
+
+For a complete guide to all properties and additional examples see [windows_firewall_profile documentation](https://docs.chef.io/resources/windows_firewall_profile)
+
+## Resource Updates
+
+### build_essential
+
+Log output has been improved in the `build_essential` resource when running on macOS systems.
+
+### chef_client_scheduled_task
+
+The `chef_client_scheduled_task` resource no longer sets up the schedule task with invalid double quoting around the command to run. Thanks for reporting this issue [@tiobagio](https://github.com/tiobagio/).
+
+### execute
+
+The `user` property in the `execute` resource can now accept user IDs as Integers.
+
+### git
+
+The `git` resource will no longer fail if syncing a branch that already exists locally. Thanks for fixing this [@lotooo](https://github.com/lotooo/).
+
+### macos_user_defaults
+
+The `macos_user_defaults` has received a ground-up refactoring with new actions, additional properties, and better overall reliability:
+
+- Improved idempotency by properly loading the current state of domains.
+- Improved how we set `dict` and `array` type data.
+- Improved logging to show the existing key/value pair that are being changed and improved the property state data we send to handlers and/or Chef Automate.
+- Fixed a failure when setting keys or values that included a space.
+- Replaced the existing non-functional `global` property with a new default for the `domain` property. To set a key/value on the `NSGlobalDomain` domain you can either set that value explicitly or just skip `domain` property entirely and Chef Infra Client will default to `NSGlobalDomain`. The existing property has been marked as deprecated and we will ship a Cookstyle rule to detect cookbooks using this property in the future.
+- Fixed the type property to only accepts valid inputs. Previously typos or otherwise incorrect values would just be ignored, resulting in unexpected behavior. This may cause failures in your codebase if you previously used incorrect values, and we will be shipping a Cookstyle rule to detect and correct these values in the future.
```

Comment:
```suggestion
- Fixed the `type` property to only accept valid inputs. Previously typos or otherwise incorrect values would just be ignored resulting in unexpected behavior. This may cause failures in your codebase if you previously used incorrect values. We will be shipping a Cookstyle rule to detect and correct these values in the future.
```